image_proc/src/crop_decimate.cpp

```diff
@@ -46,7 +46,11 @@ void debayer2x2toBGR(
   int R, int G1, int G2, int B)
 {
   typedef cv::Vec<T, 3> DstPixel;  // 8- or 16-bit BGR
-  dst.create(src.rows / 2, src.cols / 2, cv::DataType<DstPixel>::type);
+  #if CV_MAJOR_VERSION >= 3 && CV_MINOR_VERSION > 2
```

Comment:
Does the minor version actually matter? just curious.

Reply:
Yes, or put another way; the change was made in 3.3.1 it seems, https://github.com/ros-perception/image_pipeline/pull/302

Reply:
I mean I think this is fine